File: assets/js/areaofcircle.js
// Problems: Area of a circle, where the value of radius is given by the user.

/** Area of circle formula 
 *  A = πr²;
*/

const PI = 3.14159;

let r = +prompt('Enter your positive radius number: ');

// Formula A = πr²;
const A = PI * (r * r);

// Display the area of a circle
console.log('Area of the circle = ', A.toFixed(2));

File: assets/c/area-of-circle.c
// Problems: Area of a circle, where the value of radius is given by the user.

/** Area of circle formula 
 *  A = πr²;
*/

#include <stdio.h>

int main() {
  float PI = 3.14159;
  long int r, A;

  // Get input from user
  printf("Enter the radius value: ");
  scanf("%ld", &r);

  // Area of circle 
  A = PI * (r * r);

  // Display the result
  printf("Are of circle = %ld", A);

  return 0;
}